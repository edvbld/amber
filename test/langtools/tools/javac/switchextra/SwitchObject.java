/*
 * @test /nodynamiccopyright/
 * @bug 8206986
 * @compile/fail/ref=SwitchObject.out -XDrawDiagnostics SwitchObject.java
 */
public class SwitchObject {

    private int longSwitch(Object o) {
        switch (o) {
            case -1: return 0;
            case "": return 1;
            default: return 3;
        }
    }

}