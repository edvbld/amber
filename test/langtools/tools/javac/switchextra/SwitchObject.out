SwitchObject.java:8:16: compiler.err.switch.invalid.type: java.lang.Object
1 error
